src/sqllogictest/src/bin/sqllogictest.rs

```diff
@@ -154,7 +154,9 @@ async fn main() -> ExitCode {
     };
 
     if let (Some(shard), Some(shard_count)) = (args.shard, args.shard_count) {
-        eprintln!("Shard: {}/{}", shard + 1, shard_count);
+        if shard != 0 || shard_count != 1 {
```

Comment:
Shouldn't `shard_count != 1` be sufficient?

Reply:
Someone could set shard to a non-sensical value and then no tests would be run, so we should at least print out the weird values.